src/Hook/EnableDisable.php

```diff
@@ -64,6 +69,10 @@ private function onBeforeEach(BeforeEach $event)
             $request->setEnabled(false);
         }
 
+        if (count($this->bag->get('enable')) == 0) {
```

Comment:
Needed this condition from the beginning. If not requests are provided in the 'enable' data, all of them would be disabled. I apologise for missing this.